kornia/augmentation/_2d/base.py

```diff
@@ -35,8 +35,9 @@ def validate_tensor(self, input: Tensor) -> None:
 
     def transform_tensor(self, input: Tensor) -> Tensor:
         """Convert any incoming (H, W), (C, H, W) and (B, C, H, W) into (B, C, H, W)."""
+        input = _transform_input(input)
         _validate_input_dtype(input, accepted_dtypes=[float16, float32, float64])
-        return _transform_input(input)
+        return input
```

Comment:
I made a mistake with something here, will check later

Reply:
what is happening here? should not the tensor be transformed?

Reply:
This is fine as it was... i misread the `_validate_input_dtype` with `self.validate_tensor`

I'll check soon if it's working on the sequential container, which is i got the original issue from